react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import { TopicCard } from './TopicCard';
+
+type Prop = {
+  topics: { id: number; title: string; content: string }[];
```

Comment:
Since you will want to use the type `{ id: number; title: string; content: string }` in several places (such as in TopicCard), it would be best to create a type for it, and export the type. Then you can import the type in the other files where you use it. (You can export and import types the same as you do for functions.)